### PR TITLE
Add jfltexx (Samsung Galaxy S4) ass official build

### PR DESCRIPTION
**Pull request by mantinan**
Let's go for the S4

### CODE REVIEW

File: jfltexx.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "response": [
+    {
+        "maintainer": "Santiago García Mantiñán (manty)",
+        "oem": "Samsung",
+        "device": "Galaxy S4 (GT-I9505, SGH-I337M, SGH-M919)",
```
Comment (gwolf2u) [inline jfltexx.json]: Try Galaxy S4 series or similar
This is too long name

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: jfltexx.json (hunk 1 above)
Comment (mantinan) [reply jfltexx.json]: I did push a couple of changes and pushed them, don't know if I have to update the pull or similar :-?

File: jfltexx.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "response": [
+    {
+        "maintainer": "Santiago García Mantiñán (manty)",
+        "oem": "Samsung",
+        "device": "Galaxy S4 (jfltexx)",
```
Comment (gwolf2u) [inline jfltexx.json]: device codename is not needed here as it's already set by device json file

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: jfltexx.json (hunk 2 above)
Comment (mantinan) [reply jfltexx.json]: Ok, I used the two typical model names in short form this time ;-)

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**